portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const SPOTIFY_LINK = "https://open.spotify.com/playlist/6rmiBWfdA4jGdTWaYMAQri";
+
+
+/**
+ * Sends user a spotify link
```

Comment:
End this with a period.
Since its short, you can also fit this into a single line.
```
/** Sends user a spotify link. */